integration-test/src/test/java/org/jboss/pnc/integration_new/setup/RestClientConfiguration.java

```diff
@@ -34,7 +34,7 @@ public static Configuration getConfiguration(AuthenticateAs authAs) {
         if (AuthenticateAs.SYSTEM_USER.equals(authAs)) {
             builder.basicAuth(new Configuration.BasicAuth("system", "system.1234"));
         } else if (AuthenticateAs.USER.equals(authAs)) {
-            builder.basicAuth(new Configuration.BasicAuth("user", "pass.1234"));
+            builder.basicAuth(new Configuration.BasicAuth("demo-user", "pass.1234"));
```

Comment:
Why did you change this?

Reply:
I had to change authentication to "demo-user" because as old "user" when building inside the code I would have User as null because we don't have user with username:"user" in our demo-data.